### PR TITLE
added ability to pass extra flags to entrypoint

### PR DESCRIPTION
**Pull request by AndrewChubatiuk**
## What type of PR is this? 
Added ability to pass extra cmd flags to redash commands

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] New Query Runner (Data Source) 
- [ ] New Alert Destination
- [ ] Other

## Description
adding ability to pass extra arguments to entry point script, to be able to configure gunicorn, supervisord and others cmd options, besides ones, that are specified

## How is this tested?

- [ ] Unit tests (pytest, jest)
- [ ] E2E Tests (Cypress)
- [x] Manually
- [ ] N/A

### CODE REVIEW

**Comment by justinclift** [comment]
Not a bad idea.  Please fill out the details in the Issue comment thing at the top though. :smile:

**Comment by AndrewChubatiuk** [comment]
> Not a bad idea. Please fill out the details in the Issue comment thing at the top though. 😄

Done

File: compose.yaml [hunk 1]
```diff
@@ -8,10 +8,11 @@ services:
       - postgres
       - redis
     ports:
-      - "${REDASH_PORT:-5001}:5000"
+      - "${REDASH_PORT:-5001}:${REDASH_PORT:-5001}"
```
Comment (justinclift) [inline compose.yaml:11]: Why the port number change here and below?

Comment (AndrewChubatiuk) [reply compose.yaml:11]: it also happens in entrypoint, otherwise it will not work.

Comment (justinclift) [reply compose.yaml:11]: Sure, I understand that.  I'm asking why it's in this PR? :smile:

**Comment by AndrewChubatiuk** [comment]
found it's possible to pass extra args using GUNICORN_CMD_ARGS env variable. closing this issue

**Pull request closed without merging**